apps/studio/components/interfaces/Integrations/Wrappers/EditWrapperSheet.tsx

```
@@ -152,50 +154,54 @@ export const EditWrapperSheet = ({
             const hasChanges = hasFormChanges || hasTableChanges
             hasChangesRef.current = hasChanges
 
-            const encryptedOptions = wrapperMeta.server.options.filter((option) => option.encrypted)
-
             // [Alaister] although this "technically" is breaking the rules of React hooks
             // it won't error because the hooks are always rendered in the same order
             // eslint-disable-next-line react-hooks/rules-of-hooks
             useEffect(() => {
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)